tests/python/test_transformer_engine.py

```diff
@@ -51,15 +63,6 @@ def test_transformer_layer(mpi_test, benchmark, compute_type):
     rank = mpi_test.rank
 
     torch.cuda.set_device(rank)
-    os.environ["MASTER_ADDR"] = "localhost"
-    # The default port as used by https://github.com/pytorch/pytorch/blob/45a8b5682eb69d865cbf68c7f2f689b56b4efd53/torch/csrc/distributed/c10d/TCPStore.hpp#L51.
-    os.environ["MASTER_PORT"] = "29500"
-    dist.init_process_group(
-        backend="nccl",
-        init_method="env://",
-        world_size=size,
-        rank=rank,
-    )
     tp_group = dist.new_group()
```

Comment:
why do we need to create a new group here? The default pg should work as intended

Reply:
You are right. I will need that when working on 2D sharding, but I'm happy to YAGNI for now.